keras/legacy/interfaces.py

```diff
@@ -89,6 +93,33 @@ def raise_duplicate_arg_error(old_arg, new_arg):
     allowed_positional_args=['stddev'],
     conversions=[('sigma', 'stddev')])
 
+
+def lstm_args_preprocessor(args, kwargs):
+    converted = []
+    if 'forget_bias_init' in kwargs:
+        if kwargs['forget_bias_init'] == 'one':
+            _ = kwargs.pop('forget_bias_init')
+            kwargs['unit_forget_bias'] = True
+            converted.append(('forget_bias_init', 'unit_forget_bias'))
+        else:
+            _ = kwargs.pop('forget_bias_init')
+            warnings.warn('The `forget_bias_init` argument'
```

Comment:
Missing a space between "argument" and "has". Also tell users to use the `unit_forget_bias` argument instead.